cljfmt/src/cljfmt/core.cljc

```diff
@@ -248,8 +248,30 @@
 (defn- reader-conditional? [zloc]
   (and (reader-macro? zloc) (#{"?" "?@"} (-> zloc z/down token-value str))))
 
+(defn- reader-conditional-form-symbol
+  "Given a zipper pointing to a reader conditional form like
+
+    #?(:clj potemkin.core/defprotocol+ :cljs defprotocol)
+
+  Return the first symbol inside it e.g.
+
+    potemkin.core/defprotocol+"
+  [zloc]
+  (when (reader-conditional? zloc)
+    ;; find the first keyword e.g. `:clj`
+    (when-let [key-loc (z/find (-> zloc z/down z/right z/down)
+                               z/right
+                               #(n/keyword-node? (z/node %)))]
```

Comment:
Maybe instead of the comment, we use a more descriptive function:

```clojure
(when-key [key-zloc (-> zloc z/down z/right z/down find-next-keyword)]
```

Reply:
done